src/AcceptanceTesting/Common/ProviderPayments.TestStack.Infrastructure/packages.config

```diff
@@ -1,5 +1,6 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
+  <package id="AutoMapper" version="5.1.1" targetFramework="net45" />
```

Comment:
Is this package required? Automapper is something we've generally avoided here.

Reply:
It was added as the project doesn't build without it.
I've just checked and it appears to be used. Further investigation might find that it's not actually required, but I'd suggest that deeper investigation is outside of the scope of this fix.